tests/Integration/Behaviour/Features/Scenario/Product/duplicate_product.feature

```diff
@@ -112,6 +115,36 @@ Feature: Duplicate product from Back Office (BO).
     And product product1 should have following seo options:
       | redirect_type   | 301-product |
       | redirect_target | product2    |
+    When I add a specific price specific_price1 to product product1 with following details:
+      | price           | 0.00   |
+      | reduction type  | amount |
+      | reduction value | 5.00   |
+      | includes tax    | true   |
+      | from quantity   | 1      |
+    Then product "product1" should have 1 specific prices
+    When I add product product_with_combinations with following information:
+      | name[en-US] | Jar of sand  |
+      | type        | combinations |
+    And I generate combinations for product product_with_combinations using following attributes:
+      | Color | [Red,Blue] |
+    Then product "product_with_combinations" should have following combinations:
+      | id reference                   | combination name | reference | attributes    | impact on price | quantity | is default |
+      | product_with_combinationsRed   | Color - Red      |           | [Color:Red]   | 0               | 0        | true       |
+      | product_with_combinationsBlue  | Color - Blue     |           | [Color:Blue]  | 0               | 0        | false      |
+    When I add a specific price specific_price1 to product product_with_combinations with following details:
+      | price           | 123.00                       |
+      | combination     | product_with_combinationsRed |
+      | reduction type  | amount                       |
+      | reduction value | 0.00                         |
+      | includes tax    | true                         |
+      | from quantity   | 1                            |
+    And I add a specific price specific_price2 to product product_with_combinations with following details:
+      | price           | 0.00   |
+      | reduction type  | amount |
+      | reduction value | 5.00   |
+      | includes tax    | true   |
+      | from quantity   | 1      |
+    Then product "product_with_combinations" should have 2 specific prices
```

Comment:
Nice set of tests! Maybe you can put them in a separate scenario though? Because this one is already big 😅

Reply:
@atomiix Fair enough, moved the product creation to the related scenario. https://github.com/PrestaShop/PrestaShop/pull/26425/commits/f250e79e81bf947e56d42ea6c8b416a7d6bfd6b2